### PR TITLE
add circle-cross icon

### PR DESCRIPTION
**Pull request by mittalyashu**
| Preview |
| --- |
| CleanShot 2021-05-01 at 15 16 56@2x |

### CODE REVIEW

File: icons/circle-cross.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M12 22c5.523 0 10-4.477 10-10S17.523 2 12 2 2 6.477 2 12s4.477 10 10 10z" />
```
Comment (ericfennis) [inline icons/circle-cross.svg:12]:
```suggestion
  <circle cx="12" cy="12" r="10" />
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
@mittalyashu Nice icon!

Small code improvement

**Comment by locness3** [comment]
Use case?

**Comment by mittalyashu** [comment]
Use cases

* Game over
* No entry / do not enter

@locness3

**Comment by locness3** [comment]
I'm not sure about it but I think Lucide is suited for user interface icons. And I don't really see in what kind, context of UI this icon could fit.

**Comment by mittalyashu** [comment]
Okay, I will close this PR.

**Pull request closed without merging**